app/index.html

```diff
@@ -25,8 +25,53 @@
     </div>
     <div id="container" style="display: none" data-bind="visible: true,
                                                          css: { minimal: minimalView }">
+
+      <div class="dialog-overlay" data-bind="visible: connectDialog.visible() && config.defaults.easyMode">
```

Comment:
Either move this out of `defaults`, e.g. directly to `config.easyMode` or make it overwriteable via query parameters (similar to the `joinDialog` option, sometimes called `joinOnly`).
That's the rule for stuff in `defaults`: https://github.com/Johni0702/mumble-web/blob/ccdf93f6194894ef4c654b9f2a08efab61fab373/app/config.js#L27-L28